neptune/new/internal/init/model.py

```diff
@@ -43,9 +44,12 @@
 from neptune.new.metadata_containers import Model
 from neptune.new.types.mode import Mode
 
+_logger = logging.getLogger(__name__)
```

Comment:
We should use logger from `neptune.new.internal.utils.logger` created by Kasprzyk, shouldn't we?

Reply:
I think logger created by Kasprzyk is just responsible logging captured stdout/stderr to neptune.
Patter used by my is used in other parts of the code as well.

Reply:
> I think logger created by Kasprzyk is just responsible logging captured stdout/stderr to neptune.
I don't understand what you mean. We should capture all logs. What is the difference?

> Patter used by my is used in other parts of the code as well.
Yeah, because Piotrek replaced only usages of `click` for now. Still our goal should be to be consistent in whole code base.